apps/prairielearn/elements/pl-file-editor/pl-file-editor.mustache

```diff
@@ -66,7 +66,7 @@ $(function() {
         <div class="card-footer d-flex">
             <div class="ms-auto">
                 <button type="button" class="btn btn-outline-secondary btn-sm restore-original">Restore original file</button>
-                <div class="restore-original-confirm-container" style="display: none;">
+                <div class="restore-original-confirm-container d-none">
```

Comment:
Please please please test your changes. This breaks the "restore original" interaction.

Reply:
Reverted in [b9ef577](https://github.com/PrairieLearn/PrairieLearn/pull/11939/commits/b9ef57768993b494b5fe0ff9e036785845150216). https://html-validate.org/rules/no-inline-style.html#allowedproperties already had a carveout for `display: none`